### PR TITLE
feat: support to multiple pixels

### PR DESCRIPTION
**Pull request by IgorHalfeld**

### CODE REVIEW

File: packages/nuxt-meta-pixel/playground/nuxt.config.ts [hunk 1]
```diff
@@ -1,5 +1,10 @@
 export default defineNuxtConfig({
   modules: ['../src/module'],
-  nuxtMetaPixel: {},
+  nuxtMetaPixel: {
+    pixelId: '415215247513663',
+    pixels: [
+      { pixelId: '415215247513664', routes: ['/about/**'] }
+    ]
+  },
```
Comment (tanukijs) [inline packages/nuxt-meta-pixel/playground/nuxt.config.ts]: I've seen this way to do on other libs but i'm not a huge fan of having multiple ways to define pixel.s. What do u think about using a single `pixels` array even if we don't use route matching ?

We could allow something like:
```js
{
  nuxtMetaPixel: {
    pixels: [
     { id: '415215247513663' }, // match every routes
     { id: '415215247513663', routes: ['/about/**'] },
   ]
  }
}
```

File: packages/meta-pixel/package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
   "bugs": {
     "url": "https://github.com/tanukijs/meta-pixel/issues"
   },
-  "main": "lib/meta-pixel.js",
+  "main": "src/meta-pixel.ts",
```
Comment (tanukijs) [inline packages/meta-pixel/package.json:22]: Referring to the documentation [main](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#main) `lib/meta-pixel.js` was the way
But you were probably looking for [types](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package) ?

Comment (IgorHalfeld) [reply packages/meta-pixel/package.json:22]: I don't think so, I'm facing an error when running nuxt playground, It didn't happen to you?

File: packages/nuxt-meta-pixel/playground/nuxt.config.ts (hunk 1 above)
Comment (IgorHalfeld) [reply packages/nuxt-meta-pixel/playground/nuxt.config.ts]: I like the aesthetics, but how are we gonna get the default pixel if the userland adds 2 pixels without routes?

Comment (tanukijs) [reply packages/nuxt-meta-pixel/playground/nuxt.config.ts]: (discussed on discord)

File: packages/meta-pixel/package.json (hunk 2 above)
Comment (tanukijs) [reply packages/meta-pixel/package.json:22]: (discussed on discord)

**Comment by tanukijs** [comment]
Refer to #1

**Pull request merged**